.github/workflows/kserve_kind_cni_test.yaml

```diff
@@ -0,0 +1,59 @@
+name: Build & Apply KServe manifests in KinD, using istio CNI
+on:
+  workflow_dispatch:
```

Comment:
There should be a trigger here based on changes to specific directories, not just workflow_dispatch.

Reply:
@tzstoyanov will check this on monday

Reply:
If I understand the trigger correctly, only people with write access will have access to trigger the workflow based on workflow_dispatch. So, not sure if it's necessary to keep

Reply:
Yes, but this would be the easiest way to run Istio CNI tests. Lets keep it at least for one release, until Istio CNI is not enabled by default. We can remove the manual trigger at any time.

Reply:
Should this test be triggered when there is a change to istio-cni-*? Can we trigger the test to ensure it runs successfully?

Reply:
Ok, that makes more sense than the current logic. Changed the trigger action, thanks Anna.